### PR TITLE
Fix main entry in package json to point to correct file

### PR DESCRIPTION
**Pull request by Apollon77**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
       "url": "https://github.com/tarruda/has/blob/master/LICENSE-MIT"
     }
   ],
-  "main": "./src",
+  "main": "src/index.js",
```
Comment (ljharb) [inline package.json:31]: ./src is already the correct file, because index and extension lookup are automatic in CJS.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Apollon77) [reply package.json:31]: Baically right that it mostly will work, but the kind of best ptractce in the world is to point a a file here ... also according to https://docs.npmjs.com/cli/v7/configuring-npm/package-json#main by default it is a file and not "./" ...

Comment (ljharb) [reply package.json:31]: You are quite incorrect; the prevailing best practice for over a decade is to not hardcode extensions or "index" anywhere in CJS paths.

Nothing about npm's docs there contradict what i'm saying - relative file paths *always* start with `.`, it's just that node has always allowed you to omit it in `main` and `bin` and `start`. It's still best not to omit it - and `./src` in node IS a file. Try `require.resolve('has/src')` in every version of node and you'll see it points to a file just fine.

Separately, npm has no authority to define best practices for fields like "main" - that's node's purview.

Comment (Apollon77) [reply package.json:31]: All fine ... your package, Ijust never saw it withoiut a filename anyhwre :-)

**Pull request closed without merging**